README.md

```diff
@@ -1,7 +1,8 @@
-# features
+# Feature Tracking and Backlog
+
 Feature tracking repo for Kubernetes releases
 
-This repo only contains issues.  These issues are umbrellas for new features added to Kubernetes.  
+This repo only contains issues. These issues are umbrellas for new features to be added to Kubernetes. A feature can take multiple releases to complete. And a feature can be tracked as backlog items before work begins.
 
```

Comment:
Yes, I agree with this - I guess some canonical examples of multiple
parties would be Cluster Turnup (Node, Lifecycle), PetSet (Storage, API),
or GPU scheduling (API, Core & Node)

On Fri, Jul 15, 2016 at 11:37 AM, Brandon Philips <notifications@github.com>
wrote:

> In README.md
> <https://github.com/kubernetes/features/pull/26#discussion_r71021113>:
>
> >  Feature tracking repo for Kubernetes releases
> >
> > -This repo only contains issues.  These issues are umbrellas for new features added to Kubernetes.
> > +This repo only contains issues. These issues are umbrellas for new features to be added to Kubernetes. A feature can take multiple releases to complete. And a feature can be tracked as backlog items before work begins.
> >
>
> @aronchick <https://github.com/aronchick> So, do you agree with saying:
> What is a feature
>
> A feature is anything that:
>
>    - A blog post would be written about after its release (ex. minikube
>    <http://blog.kubernetes.io/2016/07/minikube-easily-run-kubernetes-locally.html>,
>    PetSet
>    <http://blog.kubernetes.io/2016/07/thousand-instances-of-cassandra-using-kubernetes-pet-set.html>,
>    rkt container runtime
>    <http://blog.kubernetes.io/2016/07/rktnetes-brings-rkt-container-engine-to-Kubernetes.html>
>    - Requires multiple parties/SIGs/owners participating to complete (ex.
>    ???)
>
> Do you have examples of multi-party features I can put in there?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/features/pull/26/files/b89c30d819bfafab54996a6058810d8b4d56f419#r71021113>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADIdUOnzByLQdPAdj9t-7McBFAowbdbks5qV9NngaJpZM4JNo-Z>
> .
>